src/divert.h

```diff
@@ -5,11 +5,18 @@
 #ifndef _EMONESP_DIVERT_H
 #define _EMONESP_DIVERT_H
 
+
 #include <Arduino.h>
 #include <MicroTasks.h>
 
 #include "evse_man.h"
 
+enum divert_type {
+  _DIVERT_TYPE_UNSET = -1,
+  _DIVERT_TYPE_SOLAR = 0,
+  _DIVERT_TYPE_GRID = 1
```

Comment:
Can we remove the `_` at the start? Still would like to use the more C++ style as well, IE `Divert type::Unset` but can do that at another point

Reply:
i've renamed them. I let you change those to C++ way or if you have an example